### PR TITLE
Simplify .md folder deletion command and replace path with variable

### PR DESCRIPTION
**Pull request by ignotus666**
# Does this need translation?

- [ ] Yes 
- [* ] No 

# Changes

See title

### CODE REVIEW

File: po4a-create-all-targets.sh [hunk 1]
```diff
@@ -48,14 +48,10 @@ fi
...
 
-while IFS= read -r -d '' dir
-do
-	lang=$(basename -s .md "$dir")
-	echo "delete $lang folder"
-	cd "$PUB_DIR"
-	rm -rf "$lang" 
-	cd ../  
-done <   <(find "$PO_DIR" -mindepth 1 -maxdepth 1 -type d -print0)
+for lang in $(ls "$PO_DIR" ) ; do
```
Comment (ann0see) [inline po4a-create-all-targets.sh:51]: Wouldn’t this delete everything (=all files) in $PO_DIR? 

You could then just `rm -rf *` I‘m a bit worried here.

Comment (ignotus666) [reply po4a-create-all-targets.sh:51]: No, it takes the directory names in PO_DIR as a reference to delete them in /wiki. That way it only deletes supported language folders (if they have a .po folder they have a counterpart in /wiki). I tested it before submitting ;)

Comment (ann0see) [reply po4a-create-all-targets.sh:51]: Yes. You’re running rm -r on $PUB_DIR

File: po4a-create-all-targets.sh [hunk 2]
```diff
@@ -94,7 +94,7 @@ use_po_module () {
 while IFS= read -r -d '' dir
 do
 	lang=$(basename -s .md "$dir")
-	echo "$lang"
+	echo '$lang'
```
Comment (ann0see) [inline po4a-create-all-targets.sh]: Wait. I think you need double quotes here to allow the content of variables be shown not the string $lang

**Review by ann0see — CHANGES_REQUESTED** [review]

File: po4a-create-all-targets.sh (hunk 2 above)
Comment (ignotus666) [reply po4a-create-all-targets.sh]: Ah yes, I got a bit overenthusiastic replacing the double quotes ...

Comment (ignotus666) [reply po4a-create-all-targets.sh]: Hmmm, I'm sure I tested it before submitting though... I'll have a look tomorrow, can't check now.

Comment (pljones) [reply po4a-create-all-targets.sh]: Well, no quotes at all are required as it's just `echo` - but definitely not single quotes...

File: po4a-update-templates.sh [hunk 3]
```diff
@@ -30,13 +30,13 @@ fi
...
 if ! [ -x "$(command -v po4a)" ] ; then
-    echo "Error: please install po4a." >&2
+    echo 'Error: please install po4a.' >&2
     exit 1
 fi
 
 # Check if source document folder exists in the right place
 if [ ! -d "$SRC_DIR" ] ; then
-    echo "Error: please run this script from the root folder"
+    echo 'Error: please run this script from the root folder'
```
Comment (pljones) [inline po4a-update-templates.sh]: This should be redirected to stderr, too (like line 33).

File: po4a-update-templates.sh [hunk 4]
```diff
@@ -77,16 +77,16 @@ do
             --master "$file" \
             --master-charset "UTF-8" \
             --po "$po_file" ; then
-        echo ""
-        echo "Error updating $lang PO file for: $adoc_file"
+        echo ''
+        echo 'Error updating $lang PO file for: $adoc_file'
```
Comment (pljones) [inline po4a-update-templates.sh]: Not much of an error as it doesn't cause termination of the script.  Should it?

Also, this second line doesn't want the single quotes as it's after variable expansion.

Comment (ignotus666) [reply po4a-update-templates.sh]: Well, it continues to update the rest of the po files. Depends on whether you want it to just exit regardless or not.

I'm working on the quote drama now. Getting rid of them all except where needed.

**Review by ann0see — APPROVED** [review]

**Pull request merged**